### ISSUE TITLE
Refine events strategy in Confluence/Notion and submit CFP

### ISSUE THREAD

**Issue by astrojuanlu** [kedro-org/kedro-devrel#23]
Potentially interesting events:

- All [PyData](https://pydata.org/) conferences (and also local meetups when appropriate)
- Some [PyCon](https://pycon.org/) and [EuroPython](https://europython.eu) events (whenever there's a sufficiently large audience of data scientists, or the event is big)
- [Open Data Science Conference](https://odsc.com/)
- Other: AI Summit, JupyterCon, WebSummit, Grace Hopper Celebration, QCon, OSCON?, FOSDEM?, OSCAFEST?

Bottom line: meet our users where they are.

**Comment by astrojuanlu** [comment]
All the events I mentioned above are (at least primarily) in-person events. But we could select some fully online or hybrid events as well. For example:

- Big Data Technology Warsaw Summit

On the other hand, we'd need an event & CFP calendar so that we can plan accordingly.

**Comment by astrojuanlu** [comment]
Upcoming:

- March 9 - [Big Data Technology Warsaw Summit](https://bigdatatechwarsaw.eu/webinar-online-2023/#9th-march)
- March 13 - [PyData London CFP deadline](https://pydata.org/london2023/call-for-proposals)
- March 18 - [PyCon Lithuania CFP deadline](https://pretalx.com/pycon-lt-2023/cfp)
- April 17 - [PyCon.DE & PyData Berlin](https://2023.pycon.de/)
- May - JupyterCon in Paris and PyCon & PyData Lithuania
- June - PyData London

**Comment by yetudada** [comment]
I put in one for ODSC Europe 🥳

**Comment by astrojuanlu** [comment]
ODSC EU: https://odsc.com/europe/call-for-speakers-europe/

- Round Two for Speaker Proposals Opens – February 27, 2023 
- Deadline for Round Two Speaker Proposals – April 3, 2023
- Second Round Accepted Speakers Notification – May 1, 2023
- Event takes place - June 14-15th, 2023

**Comment by astrojuanlu** [comment]
- June - Data + AI summit https://www.databricks.com/dataaisummit/
- September - Deep Learning Indaba 2023 https://deeplearningindaba.com/2023/
- November - Web Summit Lisbon https://websummit.com/

**Comment by astrojuanlu** [comment]
> EuroPython 2023 Call for Proposals (CFP) will be open between Monday, 6 March 2023 and Sunday, 19 March 2023

https://blog.europython.eu/europython-march-2023-newsletter/

**Comment by astrojuanlu** [comment]
Seen on LinkedIn: "As part of the development and growth of the Deep Learning Indaba, the organizing committee is welcoming workshop proposals for the first time." ([source](https://www.linkedin.com/posts/2ayasalama_call-for-workshop-proposals-deep-learning-activity-7038901995444813824-msRB?utm_source=share&utm_medium=member_desktop))

> Deadline proposal submission: April 6, 2023

https://deeplearningindaba.com/2023/indaba/call-for-workshop-proposals/

**Comment by astrojuanlu** [comment]
OpenSouthCode 2023

>  The submission period is open till March 31 2023.

https://www.opensouthcode.org/conferences/opensouthcode2023

**Comment by stichbury** [comment]
Let's keep this in the upcoming sprint and use the time to create an internal tracker for conferences/events and our planning around them.

**Comment by astrojuanlu** [comment]
Done, in review

**Comment by astrojuanlu** [comment]
Closing this as completed ✔️
